src/main/scala-3/net/codingwell/scalaguice/package.scala

```diff
@@ -0,0 +1,83 @@
+/*
+ *  Copyright 2010-2014 Benjamin Lings
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package net.codingwell
+
+import com.google.inject.internal.Annotations
+import com.google.inject.util.Types.newParameterizedType
+import com.google.inject.{Key, TypeLiteral}
+import java.lang.annotation.Annotation
+import scala.reflect.{ClassTag, classTag}
+
+
+package object scalaguice {
+
+  /**
+   * Create a [[https://google.github.io/guice/api-docs/5.0.1/javadoc/com/google/inject/TypeLiteral.html com.google.inject.TypeLiteral]] from a [[scala.reflect.runtime.universe.TypeTag]].
```

Comment:
This is no longer true. We create the TypeLiteral using Scala 3 macros.

Reply:
Fixed.